docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -34,6 +34,13 @@ Download and run the installer `.exe` file. It will install and open Teleport Co
 further user input.
 
 Repeat the process with newer versions to upgrade.
+
+To perform a silent install, that is install Teleport Connect without UI interaction, run the
+installer with the `/S` flag:
```

Comment:
I wouldn't say it requires the command line. I think the typical use case is system administrators setting up some scripts on work computers which automatically install Connect.

But this made me realize that "without UI interaction" is not necessarily correct. The only difference between silent and regular install is that the silent installation won't bring up the progress bar and won't launch Connect after the installation is complete.

How about this:

> A silent installation can be performed by running the installer with the `/S` flag. This will hide the progress bar and skip the launch of the app after the installation is complete.

Reply:
I don't like it... I LOVE IT.